TTHAnalysis/macros/leptons/LepMVAEfficiencies/README.md

```diff
@@ -0,0 +1,55 @@
+## Produce lepton MVA efficiency maps
+
+#### Content
+
+- `lepTnPFriendTreeMaker.cc`: TTree::MakeClass based macro that processes trees produced with the `treeProducerSusyMultiLepton` analyzer. Selects dilepton events (see `SelectEvent` method) consistent with DY, then looks for tag leptons passing a given selection (see `SelectTagElectron`, `SelectTagMuon` methods). For each found tag lepton, looks for a probe lepton such that the pair passes a given selection (see `SelectPair`). The tree is then filled with a few properties of the tag lepton and the pair, and all necessary properties of the probe lepton to be flexible enough to calculate efficiencies.
```

Comment:
we should also see how much of this, in general, can be done with standard mcPlots output + a fitter running on top of it